plugins/source/aws/resources/services/applicationautoscaling/scheduled_actions_mock_test.go

```diff
@@ -0,0 +1,38 @@
+package applicationautoscaling
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/applicationautoscaling"
+	"github.com/aws/aws-sdk-go-v2/service/applicationautoscaling/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client/mocks"
+	"github.com/cloudquery/plugin-sdk/faker"
+	"github.com/golang/mock/gomock"
+)
+
+func buildScheduledActions(t *testing.T, ctrl *gomock.Controller) client.Services {
+	m := mocks.NewMockApplicationautoscalingClient(ctrl)
+	services := client.Services{
+		Applicationautoscaling: m,
+	}
+	c := types.ScheduledAction{}
+	if err := faker.FakeObject(&c); err != nil {
+		t.Fatal(err)
+	}
+	output := &applicationautoscaling.DescribeScheduledActionsOutput{
+		ScheduledActions: []types.ScheduledAction{c},
+	}
+	m.EXPECT().DescribeScheduledActions(gomock.Any(), gomock.Any(), gomock.Any()).Return(
+		output,
+		nil,
+	)
+
+	return services
+}
+
+func TestScheduledActions(t *testing.T) {
+	client.AllNamespaces = []string{"test-namespace"} // Just one
```

Comment:
Is there any way we could avoid changing this? Maybe it should be a property on the client instead.

As long as `AllNamespaces` is only used by this package and all the tests here set it to the same value, it should be okay, but as soon as one of those assumptions change we could have a fairly hard-to-find bug in our testing.

I see it was like this in the code before already, so consider this comment non-blocking.